jekyll/_cci2/deploy/configure-deploy-markers.adoc

```diff
@@ -45,17 +234,49 @@ jobs:
       ...
       (existing deployment commands)
       ...
-      - run: circleci run release log --environment-name=<some-environment-name> --component-name=<some-component-name> --target-version=<some-version-name>
+      - run: circleci run release log --target-version=<some-version-name>
 ----
 
-Substitute the placeholders above based on your application's details:
+This command supports the same optional parameters as the `release plan` command, but doesn't require a `deploy-name`.
+You can see an example of the command with all optional parameters in the example below.
```

Comment:
```suggestion
You can see an example of the command with all optional parameters in the following example:
```

Reply:
Would you be ok with changing this to "You can see the command with all optional parameters in the following example:" to avoid the repetition of example?

Reply:
Oh yes definitely!